### PR TITLE
Ipfs link on justifications

### PR DESCRIPTION
**Pull request by delfipolito**
image

### CODE REVIEW

File: src/components/Proposals/ProposalDetails/ProposalDetails.js [hunk 1]
```diff
@@ -152,7 +158,27 @@ function Details({ vote, status }) {
...
+              >
+                <span
+                  css={`
+                    display: block;
+                    overflow: hidden;
+                    text-overflow: ellipsis;
+                    text-align: left;
+                  `}
+                >
+                  {getIpfsCidFromUri(ipfsJustification)}
```
Comment (Evalir) [inline src/components/Proposals/ProposalDetails/ProposalDetails.js:176]: Should we show "Read more" instead of the IPFS URI?

Comment (delfipolito) [reply src/components/Proposals/ProposalDetails/ProposalDetails.js]: Done :)

File: src/lib/web3-utils.js [hunk 2]
```diff
@@ -25,9 +25,11 @@ export function getAppPresentation(apps, appAddress) {
...
     ({ address }) => address === appAddress
   )
-
-  const iconPath = manifest.icons[0].src
-  const iconSrc = getIpfsUrlFromUri(contentUri) + iconPath
+  let iconSrc = ''
+  if (manifest.icons && manifest.icons[0]) {
+    const iconPath = manifest.icons[0].src
+    iconSrc = getIpfsUrlFromUri(contentUri) + iconPath
+  }
```
Comment (andy-hook) [inline src/lib/web3-utils.js:32]: Is this change supposed to be here? I can't see how it applies.

Comment (delfipolito) [reply src/lib/web3-utils.js:32]: Yes, sorry i put it here cause i change the org to the one that we will be  using and some of them doesnt have manifest.icons so it was breaking

**Review by andy-hook — APPROVED** [review]

**Pull request merged**